### DISCUSSION TITLE
AI教學新聞

### DISCUSSION THREAD

**Discussion by Rebecca111419** [1101-GS4719-C/Python-Community-GS4719-C#1704] [General]
今天我看到一則10月底的報導，讓我想到的確現在我自己很依賴chat gpt，特別是詢問程式，雖然它常常出錯，但至少都能提共大約的程式方向，讓我能方便思考去延伸。而看到現今小學已開始結合AI教學，看起來能解決許多教學、學習上的困難，覺得非常有趣。以下整理、擷取部分內容

**AI功能3+2，臺北酷課雲「酷AI（CooC AI）」正式上線，適性幫助每個孩子學得更好！| 臺北市政府教育局  112年10月26日**
資訊科技日新月異，臺北市政府教育局因應全球新科技浪潮，讓臺北酷課雲結合AI技術再升級！推出全新整合型AI智慧教學輔助系統「酷AI」，透過AI技術分析學生學習數據，產生個別學生學習特徵模型，提供更個別化的學習診斷與教學方針。
學生學習方面:
1.	「AI學習小幫手」創造學生與AI對話機會，引導學生學習提問的方法並享受探究學習的過程，有效提升學生探索學習及自主思辨能力
2.	「AI口說練習」學生可藉由上傳英語口說音檔，由AI提供口說表現結果分析及精進建議，可以幫助學生有效增加在語言學習中聽與說的關鍵能力
3.	「AI學習加油站」則是讓AI陪伴每位學生學習，透過分析學生各項線上學習行為，如「教材瀏覽點擊數」、「作業完成時間」及「題目作答正確率」等，即時了解學生學習情形，經AI模型演算分析後，提供學習輔助建議，並主動推播適性化的教材內容，有效率的引導學生自主學習。
在輔助教師教學方面:
1.	「AI輔助出題」協助教師編寫及發想多元評量題目，讓教師更有效率的運用教學時間
2.	「AI輔助作文批閱」則可以幫助教師快速檢視學生作文練習成果，減少評量題目訂定及作業批改時間，讓教師可以將更多心力用來關心每一位孩子的學習狀況。
　教育局表示，結合人工智慧AI技術推出的教學輔助系統-酷AI(CooC AI)，秉持以學生為中心精神，透過科技輔助有效引導學生踏上自主學習的道路，將帶領臺北市的教育邁向新的里程碑，透過AI技術分析學生學習數據，產生個別學生學習特徵模型，提供更個別化的學習診斷與教學方針，將有效落實適性學習。期待酷AI能幫助每一位孩子掌握自己的學習節奏並學得更快更好。

**Comment by Chengbosco** [comment]
我正在用gpt寫普物的期中報告，我和gpt程式的比例大概1:1吧，只能說它比我強多了，關於它寫的np.array向量陣列我到現在還沒搞懂，AI確實幫助我們學習很多。

**Comment by mrweixd** [comment]
AI超級好用，可以運用在教育、醫療、製成等。從開拓視野到精準抓錯，省去了人力的成本及提高準確率。怪不得成為主流。

**Comment by THH1118** [comment]
Chat gpt的確真的很好用，現在自己也有很多作業報告會稍微用到，但都要求自己真的想不出來了才去問，不然很怕自己以後過度依賴
：（

**Comment by 12Jesse24** [comment]
Chat GPT真的幫助我們學到很多東西，之前用javafx寫遊戲的時候多半未知的語法都是ChatGPT產出的，接著再繼續問ChatGPT
各句的程式邏輯所代表的意思，真的讓我獲益良多。
